src/web/mjs/connectors/ReadM.mjs

```diff
@@ -0,0 +1,86 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ReadM extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'readm';
+        super.label = 'Read M';
+        this.tags = [ 'manga', 'english' ];
+        this.url = 'https://readm.org';
+    }
+
+    async _getMangas() {
+        let mangas = [];
+        let index = ['', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];
+
+        for(let letter of index) {
+            let url = this.url + '/manga-list/' + letter;
+
+            let request = new Request(url, this.requestOptions);
+            let data = await this.fetchDOM(request, '.poster > a');
+
+            for (let manga of data) {
+                mangas.push(
+                    {
+                        id: this.url+manga.pathname,
+                        title: manga.querySelectorAll('h2')[0].innerText.trim()
+                    }
+                );
+            }
+        }
+
+        return mangas;
+    }
+
+    async _getChapters(manga) {
+        let chapters = [];
+
+        let request = new Request(manga.id, this.requestOptions);
```

Comment:
May use `new URL(manga.id, this,url)` in case the manga.id is a relative URL

Reply:
Is there a benefit in using `new URL(...)` over `this.url+manga.id`?

Reply:
From users view: no
From developers view: The URL object is preferred over UVstrings in case of further processing (e.g. changing paramters)

Reply:
ok, fair point. will incorporate `new URL()`